src/brogue/Architect.c

```diff
@@ -3694,9 +3689,13 @@ void initializeLevel() {
     if (getQualifyingGridLocNear(&downLoc, levels[n].downStairsLoc, grid, false)) {
         prepareForStairs(downLoc.x, downLoc.y, grid);
     } else {
-        getQualifyingLocNear(&downLoc, levels[n].downStairsLoc, false, 0,
+        boolean hasQualifyingLoc;
```

Comment:
A small thing: change to
```c
boolean hasQualifyingLoc = getQualifyingLocNear(...);
```
assignment at initialization is a little more clear and idiomatic for modern C, rather than splitting the declaration from the first assignment

Reply:
Sure, can do, I did it this way since I preferred the way the formatting looked but that's a bad reason